lib/Core/SQLiteBuildDB.cpp

```diff
@@ -109,13 +109,13 @@ class SQLiteBuildDB : public BuildDB {
 
       // Create the info table.
       if (result == SQLITE_OK) {
-	result = sqlite3_exec(
+        result = sqlite3_exec(
           db, ("CREATE TABLE info ("
-	       "id INTEGER PRIMARY KEY, "
-	       "version INTEGER, "
-	       "client_version INTEGER, "
-	       "iteration INTEGER);"),
-	  nullptr, nullptr, &cError);
+               "id INTEGER PRIMARY KEY, "
+               "version INTEGER, "
+               "client_version INTEGER, "
+               "iteration INTEGER);"),
+          nullptr, nullptr, &cError);
```

Comment:
Can you put the indentation into a separate patch? Actually I am fine if you just want to fix non-code changes like this on trunk.